config/stylelint-config-carbon/plugins/scss.js

```diff
@@ -116,4 +120,19 @@ module.exports = {
     // Disallow redundant nesting selectors (&).
     'scss/selector-no-redundant-nesting-selector': true,
   },
+  overrides: [
+    {
+      files: [
+        './packages/components/**/*.scss',
+        './packages/type/**/*.scss',
+        './packages/motion/**/*.scss',
+        './packages/layout/**/*.scss',
+        './packages/grid/**/*.scss',
+        './packages/import-once/**/*.scss',
```

Comment:
Is there any way we could include the `scss/modules` directory inside of elements packages? These should be using the right sass functions, I think

Reply:
i.e. remove `type`, `motion`, `layout` and `grid`?

Reply:
Alright, I've updated the globs to only override the older files, but will still throw validation errors for the `module` folders. I will push up another commit once I've gotten these new lint errors fixed.